lib/ansible/modules/notification/snow_record.py

```diff
@@ -148,7 +170,7 @@
 import os
 import traceback
 
-from ansible.module_utils.basic import AnsibleModule, missing_required_lib
```

Comment:
I would give this since this Ansible API which will be handling missing required libraries.